docs/wallet-compliance-guide.md

```diff
@@ -0,0 +1,27 @@
+# Wallet Compliance Guide
+
+This guide contains a set of critera wallets on Flow must meet in order to be accepted and included in Flow's FCL Wallet Discovery.
```

Comment:
Would add a why here. E.g. this is done to ensure that developers and users have a seamless experience across applications, no wallet specific code is required by developers, and users can expect apps to function as expected and support FCL's open standards. 

Would also note that why services are deemed opt-in if they don't meet that criteria.